src/main/java/Duke.java

```diff
@@ -27,9 +28,22 @@ private static void echo() {
                 line = in.nextLine();
                 continue;
             }
+            if (line.startsWith("mark")) {
+                int number = Integer.parseInt(line.substring(line.length() - 1));
+                markAsDone(number);
+                line = in.nextLine();
+                continue;
```

Comment:
The continue here is redundant

Reply:
Thank you for pointing it out, I will work on it!